libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationViewContext.kt

```diff
@@ -31,6 +40,27 @@ internal class DropInNavigationViewContext(
     var routeArrowOptions: RouteArrowOptions = RouteArrowOptions.Builder(context)
         .withAboveLayerId(RouteLayerConstants.TOP_LEVEL_ROUTE_LINE_LAYER_ID)
         .build()
+
+    val onBackPressedCallback = object : OnBackPressedCallback(true) {
```

Comment:
nit: Assuming we'll move this to BackPressManager in a future PR

Reply:
What does "nit" mean?

Reply:
I leave moving this up to you. I need it here to get back press working for InfoPanel.

Reply:
nitpick :). If you did it 💯 . back press could use some ownership, do you want to own it?

Issue with backpress is it requires configuration in the hosting activity. We'll want to fix that immediately

Reply:
Since you have a clear idea how you want this to be implemented, I think you should own it.